wordcount.c

```diff
@@ -0,0 +1,35 @@
+/*Jinoo Hong*/
+/*Write a program `wordcount.c` that takes in a text file, `text.txt` and prints to screen the number of words in the file.  You may consider any non-whitespace characters surrounded by whitespace to be a word.  The end of the file will be `"########"`.  Submit only your wordcount.c file.*/
```

Comment:
Very nice job! The only thing is you didn't take tabs into consideration so the count was just slightly off